File: project/angular_project/src/app/models/metrics.ts
import { Sensor } from "./sensor";


export class Metrics {
  metrics_id: number;
  measurement: number;
  unit_measurement: string;
  timestamp: Date;
  sensor: Sensor;

  constructor() {
    this.metrics_id = 0;
    this.measurement = 0;
    this.unit_measurement = '';
    this.timestamp = new Date();
    this.sensor = new Sensor();
  }
}

File: project/src/main/java/com/example/project/entities/Metrics.java
package com.example.project.entities;

import java.io.Serializable;
import java.util.Date;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.DBRef;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;


@Data
@Document(collection = "Metrics")
public class Metrics implements Serializable {
	@Id
	private int metrics_id;
	private int measurement;
	private String unit_measurement;
	private Date timestamp;
    private Sensor sensor;
	
}